training/src/components/Table/TableComponent.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Table, TableCell, TableContainer, TableHead, TableRow, Paper, withStyles, TableBody,
+} from '@material-ui/core';
+
+const useStyles = () => ({
+  table: {
+    minWidth: 650,
+  },
+  header: {
+    color: 'grey',
+  },
+
+});
+
+function TableComponent(props) {
```

Comment:
no need to write as TableComponent just Table is enough also change the file name.
Make this as arrow function or class component.

Reply:
done